src/dropdown/dropdown.js

```diff
@@ -10,7 +10,7 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
 
   this.open = function(dropdownScope) {
     if (!openScope) {
-      $document.on('click', closeDropdown);
+      document.body.addEventListener('click', closeDropdown, true);
```

Comment:
Ah, I missed that. If that is necessary, I'd prefer `$document.find('body').on(...)`